load.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Analytics Module functions.
+ *
+ * @package hm-platform/analytics
+ */
+
+namespace HM\Platform\Workflow;
+
+use const HM\Platform\ROOT_DIR;
+use HM\Platform\Module;
+
+function bootstrap( Module $module ) {
+	$settings = $module->get_settings();
+
+	if ( $settings['google-tag-manager'] ) {
+		add_action( 'muplugins_loaded', __NAMESPACE__ . '\\load_gtm' );
```

Comment:
Any reason not to load with `plugins_loaded` instead?

Reply:
Yep, pretty common for plugins to use (and expect to be able to use) the `plugins_loaded` hook themselves so I don't think we should be loading anything from a module on that hook.